client/my-sites/people/invite-people/index.jsx

```diff
@@ -270,6 +271,18 @@ const InvitePeople = React.createClass( {
 	},
 
 	render() {
+		if ( this.props.site && ! get( this.props, 'site.capabilities.promote_users' ) ) {
```

Comment:
You should use the userCan utility function here. https://github.com/Automattic/wp-calypso/blob/master/client/lib/site/utils.js#L7